src/EnergyPlus/GroundTemperatureModeling/GroundTemperatureModelManager.hh

```diff
@@ -0,0 +1,64 @@
+#ifndef GroundTemperatureModelManager_hh_INCLUDED
+#define GroundTemperatureModelManager_hh_INCLUDED
+
+// C++ Headers
+#include <memory>
+
+// EnergyPlus Headers
+#include <EnergyPlus.hh>
+#include <GroundTemperatureModeling/BaseGroundTemperatureModel.hh>
+#include <DataGlobals.hh>
+
+namespace EnergyPlus {
+
+namespace GroundTemperatureManager {
+
+	extern int const objectType_KusudaGroundTemp;
+	extern int const objectType_FiniteDiffGroundTemp;
+	extern int const objectType_SiteBuildingSurfaceGroundTemp;
+	extern int const objectType_SiteShallowGroundTemp;
+	extern int const objectType_SiteDeepGroundTemp;
+	extern int const objectType_SiteFCFactorMethodGroundTemp;
+	extern int const objectType_XingGroundTemp;
+
+	extern Array1D_string const CurrentModuleObjects;
+
+	static std::vector < std::shared_ptr < BaseGroundTempsModel > > groundTempModels;
```

Comment:
@mmatts This might not be doing what you expect or want (maybe it is, I don't know). If you have a static variable in a header then every time you include that header you get a new, unique version of that variable. Read [this](https://stackoverflow.com/questions/92546/variable-declarations-in-header-files-static-or-not) for more information.

@DeadParrot @Myoldmopar